simulations/vip-Gateway/bsctestnet.ts

```diff
@@ -0,0 +1,208 @@
+import { TransactionResponse } from "@ethersproject/providers";
+import { expect } from "chai";
+import { BigNumber, Signer } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents, initMainnetUser } from "../../src/utils";
+import { NORMAL_TIMELOCK, forking, testVip } from "../../src/vip-framework";
+import {
+  COMPTROLLER_BEACON,
+  NATIVE_TOKEN_GATEWAY,
+  NEW_COMPTROLLER_IMPLEMENTATION,
+  NEW_VTOKEN_IMPLEMENTATION,
+  VTOKEN_BEACON,
+  vipGateway,
+} from "../../vips/vip-Gateway/bsctestnet";
+import BEACON_ABI from "./abi/Beacon.json";
+import COMPTROLLER_ABI from "./abi/Comptroller.json";
+import ERC20_ABI from "./abi/ERC20.json";
+import NATIVE_TOKEN_GATEWAY_ABI from "./abi/NativeTokenGateway.json";
+import VTOKEN_ABI from "./abi/VToken.json";
+
+const OLD_COMPTROLLER_IMPLEMENTATION = "0x329Bc34E6A46243d21955A4369cD66bdD52E6C22";
+const OLD_VTOKEN_IMPLEMENTATION = "0xe21251bc79ee0abeba71faabdc2ad36762a0b82f";
+
+const VWBNB_LIQUID_STAKED_BNB = "0x231dED0Dfc99634e52EE1a1329586bc970d773b3";
+const VBNBX_LIQUID_STAKED_BNB = "0x644A149853E5507AdF3e682218b8AC86cdD62951";
+const POOL_LIQUID_STAKED_BNB = "0x596B11acAACF03217287939f88d63b51d3771704";
+
+const BNBX = "0x327d6E6FAC0228070884e913263CFF9eFed4a2C8";
+const WBNB = "0xae13d989daC2f0dEbFf460aC112a837C89BAa7cd";
+
+const USER_1 = "0x03862dFa5D0be8F64509C001cb8C6188194469DF";
+const USER_2 = "0xb07A7f0A252bc7a3a26aa3c3C9D9c38aD8a6f02F";
+
```

Comment:
I would use the `checkCorePoolComptroller` check, to confirm the upgrade in the Core pool was ok. I would try to use also the `checkIsolatedPoolsComptrollers` check, to check the same in the IL pool

Reply:
Ok

Reply:
Done https://github.com/VenusProtocol/vips/pull/195/commits/7c5ed791622ec5b45acc6c6d09e104a3b58d88ec